src/sympc/approximations/sigmoid.py

```diff
@@ -41,3 +44,30 @@ def sigmoid(tensor: MPCTensor, method: str = "exp") -> "MPCTensor":
             result += (tensor ** d) * weights[i + 1]
 
         return result
+
+    elif method == "chebyshev":
+        # Make sure the elements are all positive
+        _sign = sign(tensor)
+        tensor_8 = tensor * _sign / 8
```

Comment:
maybe detail why you divide by 8?

Reply:
and rename tensor_8 accordingly

Reply:
> maybe detail why you divide by 8?

That's a part of way the approximation works. Check out the definition of the sigmoid function used in the image in the description.

![image](https://user-images.githubusercontent.com/31875325/116692654-55a87480-a9da-11eb-8917-bb5344c3c0e5.png)


> rename tensor_8 accordingly

Any ideas for an appropriate name? I felt that `tensor_8` was fine :sweat_smile:

Reply:
I don't understand in the taylor formula where it is said that `w` should be tensor/8 ? This is just scaling right?

Reply:
Putting down the discussion here as well. 
- Scaling issue
  - `R` = Regular sigmoid function (that we all know and love) = `1/(1+e^-x)`
  - `S` = Standard sigmoid function (defined in the paper) = `1/(1+e^-8x)`
  - Hence, `R(x)` = `S(x/8)`

- Renamed to `positive_tensor_rescaled`